app/controllers/characters_controller.rb

```diff
@@ -1,19 +1,37 @@
 class CharactersController < ApplicationController
+   before_action :set_character, only: %i[show update destroy]
+
   def index
     @characters = Character.all
     json_response(@characters)
   end
 
   def create
+    @character = Character.create!(character_params)
+    json_response(@character, :created)
   end
 
   def show
+    json_response(@character)
   end
 
   def update
+    @character.update(character_params)
+    head :no_content
```

Comment:
🤔 we don't want to return the JSON updated?

Reply:
https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/204#:~:text=The%20common%20use%20case%20is,200%20should%20be%20used%20instead.